bulk_site_creator/templates/bulk_site_creator/new_job.html

```diff
@@ -37,13 +37,13 @@ <h3 class="breadcrumbs">
                             </label>
                             <br>
                             <select id="courseDepartment" name="courseDepartment" class="form-control">
-                                {% if not selected_department_id %}
-                                    <option value="0" selected>Department</option>
+                                {% if not selected_department_id or selected_department_id == '0' %}
+                                    <option value="department:0" selected>Department</option>
                                 {% else %}
-                                    <option value="0">Department</option>
+                                    <option value="department:0">Department</option>
                                 {% endif %}
                                 {% for department in departments %}
-                                    {% if selected_department_id and selected_department_id == department.id %}
+                                    {% if selected_department_id and "dept:"|add:selected_department_id == department.id %}
```

Comment:
I realize that we're throwing away everything before the `:` but it's a little confusing to use `dept` here but `department` for the option above.

Reply:
Ah that's a good catch, those are typos. I will update